drake/systems/Simulation.h

```diff
@@ -74,11 +104,24 @@ void simulate(const System& sys, double ti, double tf,
       Eigen::VectorXd::Zero(getNumInputs(sys)));
   typename System::template OutputVector<double> y;
 
+  bool rt_warning_printed = false;
+
   // Take steps from ti to tf.
   double t = ti;
   while (t < tf) {
-    handle_realtime_factor(start, t, options.realtime_factor,
-                           options.timeout_seconds);
+    try {
+      handle_realtime_factor(start, t, options.realtime_factor,
+                             options.timeout_seconds);
+    } catch(std::runtime_error err) {
+      if (options.warn_real_time_violation) {
+        if (!rt_warning_printed) {
+          std::cerr << "WARNING: " << err.what() << std::endl;
+          rt_warning_printed = true;  // supress future warnings
```

Comment:
This comment is a sentence, so needs capitalization and punctuation (_cappunc_).